runtime/autoload/provider/pythonx.vim

```diff
@@ -54,7 +54,7 @@ function! s:check_interpreter(prog, major_ver, skip) abort
 
   " Try to load neovim module, and output Python version.
   let prog_ver = system([ a:prog , '-c' ,
-        \ 'import sys; sys.stdout.write(str(sys.version_info[0]) + '.
+        \ 'import sys; sys.path.remove(""); sys.stdout.write(str(sys.version_info[0]) + '.
```

Comment:
In this case `sys.path` does not contain `"."` and `""` denotes the current directory. From the Python docs on `sys.path`:

> If the script directory is not available (e.g. if the interpreter is invoked interactively or if the    script is read from standard input), path[0] is the empty string, which directs Python to search modules in the current directory first.